hls4ml/templates/vivado/firmware/parameters.h

```diff
@@ -3,6 +3,7 @@
 
 #include "ap_int.h"
 #include "ap_fixed.h"
+#include "defines.h"
```

Comment:
is this necessary? usually both `defines.h` and `parameters.h` are included `myproject.cpp`, but separately

Reply:
Ah I see we don't need it.